synthtool/gcp/templates/node_library/.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,18 +1,52 @@
----
-name: Feature request
-about: Suggest an idea for this library
-labels: 'type: feature request, priority: p3'
----
+name: Feature Request
+description: Suggest an idea for this library
+labels:
+  - feature request
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **PLEASE READ**: If you have a support contract with Google, please
+        create an issue in the [support
+        console](https://cloud.google.com/support/) instead of filing on GitHub.
+        This will ensure a timely response. Otherwise, please make sure to
+        follow the steps below.
+  - type: textarea
+    attributes:
+      label: >
+        A screenshot that you have tested with "Try this API".
+      description: > 
+        As our client libraries are mostly autogenerated, we kindly request
+        that you test whether your feature request is with the client library, or with the
+        API itself. To do so, please search for your API
+        here: https://developers.google.com/apis-explorer and attempt to
+        reproduce the issue in the given method. Please include a screenshot of
+        the response in "Try this API". This response should NOT match the current
+        behavior you are experiencing. If the behavior is the same, it means
+        that you are likely requesting a feature for the API itself. In that
+        case, please submit an issue to the API team, either by submitting an
+        issue in its issue tracker https://cloud.google.com/support/docs/issue-trackers, or by
+        submitting an issue in its linked tracker in the .repo-metadata.json
+        file in the API under packages/* ([example]({{ metadata['repo']['issue_tracker'] }}))
 
-Thanks for stopping by to let us know something could be better!
+         Examples of common library-specific issues vs. API issues are: retry strategies, authentication questions, issues with typings. 
+    validations:
```

Comment:
The phrasing and formatting of this is kind of confusing to me - I think that a reformat and an addition of some API issues would help. Something like

```
Example of library specific issues would be: retry strategies, authentication questions, or issues with typings.
Examples of API issues would include: accepting a new file type as a parameter in a particular method, adding functionality to an API.
```

Also I'm totally spitballing for those API ones and am not attached to them at all. Feel free to ditch em